### ISSUE TITLE
title

### ISSUE THREAD

**Issue by amromeo** [amromeo/api_r2020#15]
@jwrudolf @pcmathias 
guys what is 
R Language Primer and Programming Symposium on Data Sciences?

**Comment by pcmathias** [comment]
That is the official title on the Path Informatics Summit schedule: https://www.pathologyinformatics.org/schedule_2019.php

Figured it was a fancy name you came up with. A bit wordy for my tastes.

**Comment by skadauke** [comment]
Title on title page is: Introduction to R Workshop.
